src/os/bluestore/KernelDevice.cc

```diff
@@ -548,3 +557,22 @@ int KernelDevice::invalidate_cache(uint64_t off, uint64_t len)
   return r;
 }
 
+int KernelDevice::aio_read(uint64_t off, uint64_t len, bufferlist *pbl, IOContext *ioc, bool buffered)
```

Comment:
Yeah. Other question: 
a)for read, is is ok for async mode w/o O_DIRECT?
b)If a)not, can it change read-mode from buffer to direct for wal-read?